src/search/pdbs/cegar.cc

```diff
@@ -197,24 +197,24 @@ CEGAR::CEGAR(
 }
 
 void CEGAR::print_collection() const {
-    utils::g_log << "[";
+    log << "[";
     for (size_t i = 0; i < pattern_collection.size(); ++i) {
         const unique_ptr<PatternInfo> &pattern_info = pattern_collection[i];
         if (pattern_info) {
-            utils::g_log << pattern_info->get_pattern();
+            log << pattern_info->get_pattern();
             if (i != pattern_collection.size() - 1) {
-                utils::g_log << ", ";
+                log << ", ";
             }
         }
     }
-    utils::g_log << "]" << endl;
+    log << "]" << endl;
 }
 
 bool CEGAR::time_limit_reached(
     const utils::CountdownTimer &timer) const {
     if (timer.is_expired()) {
-        if (verbosity >= utils::Verbosity::NORMAL) {
-            utils::g_log << "CEGAR time limit reached" << endl;
+        if (log.is_at_least_normal()) {
```

Comment:
We already had the bike-shedding discussion about these names, right?
It seems a long name, but I don't have an awesome alternative. And it's not what this issue is about. I'm just interested.